### DISCUSSION TITLE
login error , Please help!

### DISCUSSION THREAD

**Discussion by Tobi-maru** [prasanthrangan/hyprdots#1660] [General]
Hi ,I am using arch linux and recently i came across this hyprland desktop environment . I used it for a while but suddenly one day i am not able to log into my hprland environment . Everytime i enter my user name and password a blank screen appers and after a while the same login screen appears . I also try to start up the environment from shell but keep getting **Hyprland: symbol lookup error: Hyprland: undefined symbol: _ZNK9Hyprutils4Math4CBox13containsPointERKNS0_8Vector2DE** I thought there is a problem with my display manager so i changed it  , i even reinstalled the whole environment but still its not working and all the other environments like gnome are working fine . So if you know anything about it please help

**Comment by Tobi-maru** [comment]
![Screenshot from 2024-07-04 10-59-52](https://github.com/prasanthrangan/hyprdots/assets/98447782/2646d3e5-d042-4d1d-958d-4b68b9e42230)

**Reply by dieBakterie** [reply]
It seems there is a small mistake in the .zshrc file of yours it's `/home/potato/fzf-git.sh`or is it really **/home/potato/fzf-git.sh/fzf-git.sh**?!

**Reply by dieBakterie** [reply]
Seems to be upstream, try to use the -git version and see if the error remains. Whne using hyprland-git then use the whole -git ecosystem, hypridle-git, hyprpicker-git etc only if you use those of course, dont need to install them extra. I'm using hyprlock, hypridle etc

**Reply by Tobi-maru** [reply]
> It seems there is a small mistake in the .zshrc file of yours it's `/home/potato/fzf-git.sh`or is it really **/home/potato/fzf-git.sh/fzf-git.sh**?!

Its just the path of a tool (fuzzyfinder) I added to .zshrc file and later on I deleted the tool so thats why its showing missing directory

**Comment by kRHYME7** [comment]
Reinstall your hyprland (Like remove the package (-Rns) and install again(-Sy)  )

**Reply by Tobi-maru** [reply]
I did that its not working

**Comment by Tobi-maru** [comment]
I googled the various errors i was getting and came to know that it is some linker problem . The problem is not is the desktop env.  itself . I did some digging inside the arch directories and some of the files are corrupted .My best guess is there is some problem in the arch itself .I am thinking about reinstalling the arch again and see if the problem presist until then i will continue to use gnome .

**Comment by Tobi-maru** [comment]
Lately something weird is also happening i was writting some code in C language I imported the math lib. but i cannt use any functions included in the math lib. saying unidentified reference to functions my best guess is there is some problem with the linker 
. It is not able to link the files proberly .

**Comment by Tobi-maru** [comment]
![Screenshot from 2024-07-06 15-09-33](https://github.com/prasanthrangan/hyprdots/assets/98447782/ec650eb3-018e-4128-83ed-1ee268755e7b)
